test/test_other.py

```diff
@@ -2592,6 +2592,10 @@ def test_contrib_ports(self):
     # with a different contrib port when there is another one
     self.emcc(test_file('other/test_contrib_ports.cpp'), ['--use-port=contrib.glfw3'])
 
+  @requires_network
+  def test_remote_ports(self):
+    self.emcc(test_file('hello_world.c'), ['--use-port=contrib.emdawnwebgpu'])
```

Comment:
@kainino0x the name `emdawnwebgpu` seems rather long and redundant to me.

How about simple `emdawn` or even just `dawn`.   The `em` prefix and the `webgpu` suffix both seem redundant in this context but maybe I'm missing something?

In any case we can land this and bikeshed later.

Reply:
FWIW,  I agree 100% with @sbc100. I had named it originally `dawn` because it is the name of the project and it is following the pattern of all the other ports (`sdl2`, `glfw3`, etc... which are not named `emsdl2`, `emglfw3`). I only renamed it because @kainino0x asked me to.

Reply:
Emdawnwebgpu is the name of the project, it's used for the package releases and in the Dawn source tree. I chose it to be easily searchable and unambiguous (from USE_WEBGPU, any other potential/future ways of using Dawn in Wasm, Tint in Wasm, etc.)

Reply:
Fair enough!

Also, we could drip the `contrib.` here if you like since we do care internally about maintain this port if good working order.   When we could just have `--use-port=emdawnwebg`

Reply:
If the distinction is how much we maintain it, then making it a non-contrib port makes sense to me. Will probably roll that change into #24220.

Reply:
Yeah, kind of.  Its not super well defined.  Basically I wanted to allow folks to create new ports in emscripten without adding more work for emscripten developers to make sure they keep working in all out different build configurations.  I think we webgpu is key enough that we should consider it core and make sure it works over time.

That said, we should at some point add a few more emscripten-side tests to ensure we don't break things on our end.